Time Tracker/CSV.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using nsButton;
+using System.IO;
+
+namespace nsCSV
+{
+    class CSV : Button
+    {
+        private List<string> list = new List<string>();
+        public void WriteLogsIntoCSV()
```

Comment:
think it again I think you can do it better . Print all the text at once maybe Its not best idea

Reply:
Btw, the csu class shouldn't know what to write, only how and maybe where you want it to be written